pkg/fees/btsTxFees.test.ts

```diff
@@ -0,0 +1,6 @@
+import { getFeesBTS } from "./btsTxFees"
+ 
+test("compares the BTS fee value", async () => {
```

Comment:
Ideally this is worded something like 
it("calculates the correct value", ...
Because currently the user wouldn't know whether the "comparison" should be true or false just by reading the message.

That way it is easy to read on the console output.

Reply:
Done